auth-web/src/util/common-util.ts

```diff
@@ -273,6 +273,11 @@ export default class CommonUtils {
     return (url) ? url.trim().replace(/\/+$/, '') : ''
   }
 
+  /** use NR request action code to get NR type from enum */
+  static mapRequestActionCdToNrType (requestActionCd: NrRequestActionCodes): string {
+    return NrRequestActionStrings[requestActionCd] as string
+  }
```

Comment:
I'd rather see a static string mapping here instead of dereferencing an array with a variable.

This won't fail (it will return undefined) but it's not good practice. Ideally you would check array bounds before dereferencing, or make sure the array contains all possible values (per the enum key).

Reply:
So using something like `NrRequestActionStrings.get(requestActionCd)` with a map? I should probably also update [mapRequestTypeCdToNrType](https://github.com/bcgov/sbc-auth/blob/main/auth-web/src/util/common-util.ts#L277) since it uses the same logic as this currently.

Reply:
I was thinking that you don't need a map for a single case. I was also thinking that you'd put any caveats here.

Reply:
Have a look at my latest change and let me know if that is what you mean.

Reply:
Exactly. It's very clear now, thanks!